modules/nodes-cluster-overcommit-buffer-chunk.adoc

```diff
@@ -12,6 +12,12 @@ to switch to file buffering to reduce memory usage and prevent data loss.
 
 Fluentd file buffering stores records in _chunks_. Chunks are stored in _buffers_.
 
+[NOTE]
+====
+To modify the `BUFFER_QUEUE_LIMIT`, `FILE_BUFFER_LIMIT`, or `BUFFER_SIZE_LIMIT` parameters
```

Comment:
As mentioned in [BZ-1827182](https://bugzilla.redhat.com/show_bug.cgi?id=1827182) you a cannot modify `BUFFER_QUEUE_LIMIT` directly.